Information/Design.md

```diff
@@ -310,7 +354,7 @@ examples of _-edit_  command usage:
 ------------------------------------------------------------------------------------------
 
 
-######REQ 160 Design:
+######REQ 160 Design:  Josh
```

Comment:
Missing the word 'by' that all of the others have.

Reply:
Ok I will fix it

Reply:
Ok will fix it

Reply:
fixed it